app/components/exp-lookit-iframe/component.js

```diff
@@ -36,21 +35,21 @@ export default ExpFrameBaseComponent.extend({
     },
 
     didReceiveAttrs() {
-        const iframeSrc = this.get('frameConfig.iframeSrc');
+        // call super first in case iframeSrc comes from generateProperties
+        this._super(...arguments);
+        const iframeSrc = this.get('iframeSrc');
         const hashChildId = this.get('session.hash_child_id');
         const responseId =  this.get('session.id');
-        this.set('frameConfig.iframeSrc', addSearchParams(iframeSrc, responseId, hashChildId));
+        this.set('iframeSrc', addSearchParams(iframeSrc, responseId, hashChildId));
+    },
```

Comment:
In case anyone is wondering about the switch to modifying the `iframeSrc` value rather than `frameConfig.iframeSrc`. The `frameConfig` object contains the user-defined parameters and is used by the super class to set all of the frame's parameter values (explicitly set and defaults), which are copied into `this` as top-level properties. So _after_ calling super in `didReceiveAttrs`, `this.iframeSrc` is the iframe URL that's actually used by the frame template.